packages/react/src/components/slider/slider.tsx

```diff
@@ -0,0 +1,262 @@
+        margin-left: -16px;
+        position: absolute;
+        width: 32px;
+
+        &::before {
+            background-color: ${({ theme }) => theme.component['slider-thumb-background-color']};
+            border: 2px solid ${({ theme }) => theme.component['slider-thumb-border-color']};
+            border-radius: 50%;
+            box-sizing: border-box;
+            content: '';
+            display: block;
+            height: 24px;
+            left: 50%;
+            outline: 0;
+            position: absolute;
+            top: 50%;
+            transform: translate(-50%, -50%);
+            transform-origin: center;
+            transition-duration: 120ms;
+            transition-property: box-shadow, width, height;
+            width: 24px;
+        }
+        
+        &.${sliderClasses.active},
+        &:hover {
+            &::before {
+                height: 32px;
+                width: 32px;
+            }
+        }
+
+        &.${sliderClasses.focusVisible} {
+            &::before {
+                box-shadow: inset 0 0 0 2px ${({ theme }) => theme.component['focus-inside-border-color']};
+            }
+        }
+        
+        &.${sliderClasses.active} {
+            &::before {
+                box-shadow: inset 0 0 0 2px ${({ theme }) => theme.component['slider-thumb-border-color']};
+            }
+        }
+
+        &.${sliderClasses.focusVisible},
+        &:hover {
+            ${ValueTooltip} {
+                transform: scale(1);
+            }
+        }
+    }
+
+    & .${sliderClasses.mark} {
+        background-color: ${({ theme }) => theme.component['slider-rail-mark-background-color']};
+        border-radius: 99%;
+        height: 2px;
+        position: absolute;
+        transform: translateX(-50%);
+        width: 2px;
+    }
+
+    & .${sliderClasses.markActive} {
+        background-color: ${({ theme }) => theme.component['slider-track-mark-background-color']};
+    }
+
+    &.${sliderClasses.disabled} {
+        cursor: default;
+        pointer-events: none;
+        
+        .${sliderClasses.rail} {
+            background-color: ${({ theme }) => theme.component['slider-rail-disabled-background-color']};
+        }
+
+        .${sliderClasses.track} {
+            background-color: ${({ theme }) => theme.component['slider-track-disabled-background-color']};
+        }
+
+        .${sliderClasses.thumb} {
+            &::before {
+                border-color: ${({ theme }) => theme.component['slider-thumb-disabled-border-color']};
+            }
+        }
+
+        .${sliderClasses.mark} {
+            background-color: ${({ theme }) => theme.component['slider-rail-mark-disabled-background-color']};
+        }
+
+        & .${sliderClasses.markActive} {
+            background-color: ${({ theme }) => theme.component['slider-track-mark-disabled-background-color']};
+        }
+    }
+`;
+
+const SliderValueTooltip: FunctionComponent<SliderValueLabelSlotProps> = ({ children }) => (
```

Comment:
Pour rester compatible avec React 18
```suggestion
const SliderValueTooltip: FunctionComponent<PropsWithChildren<SliderValueLabelSlotProps>> = ({ children }) => (
```

Reply:
On ne le voit pas, mais le children est explicitement defini dans SliderValueLabelSlotProps.
https://github.com/mui/material-ui/blob/47329acc3f318c2a427c0d025d3aaf7aa6a13c66/packages/mui-base/src/Slider/Slider.types.ts#L219